slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt

```diff
@@ -396,6 +396,8 @@ class SlideToActView @JvmOverloads constructor(
 
                 mArrowMargin = mIconMargin
                 mTickMargin = mIconMargin
+
+                mIsCompleted = getBoolean(R.styleable.SlideToActView_complete, false)
```

Comment:
You should check at the end of the constructor what's the value of `mIsCompleted` and call the correct function

Reply:
Unfortunately, there is a problem.  `setCompleted(mIsCompleted) ` can't be called in constructor as some values like `mAreaWidth`, `mAreaHeight` will be set up only in `onSizeChanged(w: Int, h: Int, oldw: Int, oldh: Int)` function.

Reply:
That also works 👍